code/modules/projectiles/gun.dm

```diff
@@ -143,7 +143,7 @@
 	return !user.contains(src)
 
 /obj/item/gun/proc/shoot_with_empty_chamber(mob/living/user as mob|obj)
-	visible_message(span_warning("*click*"), vision_distance = COMBAT_MESSAGE_RANGE)
+	balloon_alert_to_viewers("*click*", vision_distance = COMBAT_MESSAGE_RANGE)
```

Comment:
Does vision_distance actually work for balloon_alert_to_viewers? I don't think it does.

Reply:
I think you should just remove the vision distance entirely since there's already a sound for it, and so you'd be able to hear it anyway I'd think.

Reply:
Just noticed @MrMelbert brought this up. Does balloon_alert_to_viewers even support this?  I don't remember adding that. Also, is there a case where you hear the sound but not see the message? Both need to happen at the same time for a11y.

Reply:
Vision distance does work. It defaults to `DEFAULT_MESSAGE_RANGE` (7). I'm fine with making this 7 instead of 3 but you'll have to fight Melbert for it.

Reply:
I just want to make sure there's no case where you can see it but not hear it or hear it and not see it.

Reply:
I switched this back to DEFAULT_MESSAGE_RANGE.